lib/ansible/modules/storage/netapp/na_ontap_cifs_server.py

```diff
@@ -0,0 +1,308 @@
+#!/usr/bin/python
+""" this is cifs_server module
+
+ (c) 2018, NetApp, Inc
+ # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+"""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: na_ontap_cifs_server
+short_description: cifs server configuration
+extends_documentation_fragment:
+    - netapp.na_ontap
+version_added: '2.6'
+author: chhaya gunawat (chhayag@netapp.com)
+
+description:
+    - Creating / deleting and modifying the CIF server .
+
+options:
+
+  state:
+    description:
+    - Whether the specified cifs_server should exist or not.
+    default: present
+    choices: ['present', 'absent']
+
+  service_state:
+    description:
+    - CIFS Server Administrative Status.
+    required: true
```

Comment:
remove this - not required (not sure why validate-modules didn't catch this discrepancy)